package.json

```diff
@@ -38,17 +38,17 @@
     "ember-fetch": "^6.2.0"
   },
   "peerDependencies": {
-    "@esri/arcgis-rest-auth": "^2.4.0",
-    "@esri/arcgis-rest-portal": "^2.4.0",
-    "@esri/arcgis-rest-request": "^2.4.0",
-    "@esri/arcgis-rest-types": "^2.4.0"
+    "@esri/arcgis-rest-auth": "^2.17.0",
+    "@esri/arcgis-rest-portal": "^2.17.0",
+    "@esri/arcgis-rest-request": "^2.17.0",
+    "@esri/arcgis-rest-types": "^2.17.0"
```

Comment:
afaict, only portal needs to be bumped here. none of the other packages changed at 2.17, right?

Reply:
> . none of the other packages changed at 2.17, right?

Correct. Looked like they were intentionally kept in sync so followed suit. Happy to revert the other packages though.

Reply:
in fact, b/c we didn't introduce any code that uses `removeUsers()`, which is the only new fn in 2.17, we should **not** bump any peerDeps at this time.